src/main/java/com/stripe/net/RequestOptions.java

```diff
@@ -115,12 +115,34 @@ public static RequestOptionsBuilder builder() {
   /**
    * Convert request options to builder, retaining invariant values for the integration.
    *
+   * @deprecated prefer {@link toBuilderFullCopy} which fully copies the request options instead of
+   *     a subset of its options.
    * @return option builder.
    */
+  @Deprecated
   public RequestOptionsBuilder toBuilder() {
     return new RequestOptionsBuilder().setApiKey(this.apiKey).setStripeAccount(this.stripeAccount);
   }
 
+  /**
+   * Convert request options to builder, copying all options.
+   *
+   * @return option builder.
+   */
+  public RequestOptionsBuilder toBuilderFullCopy() {
```

Comment:
Do we even need this one?

Reply:
Yes, because `com.stripe.net` is not the same package as `com.stripe.model`

Reply:
There might be a solution involving [JPMS](https://www.oracle.com/corporate/features/understanding-java-9-modules.html) but since we still support Java 8 I haven't done much research there.

Reply:
Can we inline the implementation into the place it's currently used at?

Reply:
No, because these fields are private

Reply:
Don't all of them have a getter?